src/ReverseInteger.java

```diff
@@ -0,0 +1,26 @@
+public class ReverseInteger {
+    public static int reverse(int x) {
+        long res = 0;
+        int mod = Math.abs(x);
+        int i = (int) Math.log10(mod);
+        if(i == 0 && (mod > 10 || mod < -10)) {
```

Comment:
What is the intention here? (mod < -10) can never be true because mod is a positive number